app/javascript/packs/ping_thing/stats_chart.js

```diff
@@ -146,7 +148,11 @@ export default {
                             callbacks: {
                                 label: function(tooltipItem) {
                                     if (tooltipItem.datasetIndex == 0) {
-                                        return "Min: " + tooltipItem.raw[1].toLocaleString('en-US') + " ms,  Max: " + tooltipItem.raw[0].toLocaleString('en-US') + " ms";
+                                        var min = tooltipItem.raw[1] ? tooltipItem.raw[1].toLocaleString('en-US') : "-";
+                                        var max = tooltipItem.raw[0] ? tooltipItem.raw[0].toLocaleString('en-US') : "-";
+                                        var slot_lat = tooltipItem.raw[2] ? tooltipItem.raw[2].toLocaleString('en-US') : "-";
+
+                                        return "Min: " + min + " ms, Max: " + max + " ms, Average_slot_latency: " + slot_lat + " slots";
```

Comment:
Let's put this string in two separate lines, it will look better:
```suggestion
                                        return ["Min: " + min + " ms, Max: " + max + " ms", "Average_slot_latency: " + slot_lat + " slots"];
```